examples/online_serving/multi-node-serving.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+subcommand=$1
+shift
+
+ray_port=6379
+ray_init_timeout=300
```

Comment:
Can those be changed a env vars?

Reply:
actually we are setting them as command line flags, resolving, so this is not necessary.